app/models/hmpps_api/sentence_detail.rb

```diff
@@ -11,14 +11,10 @@ class SentenceDetail
                 :licence_expiry_date,
                 :sentence_start_date,
                 :tariff_date,
-                :sentence_expiry_date,
                 :actual_parole_date
 
-    def initialize(fields = {})
-      fields.each do |k, v|
-        instance_variable_set("@#{k}", v)
-      end
-    end
+    # Note - this is hiding a defect - we never get sentence_expiry_date from NOMIS (but maybe we should?)
+    attr_accessor :sentence_expiry_date
```

Comment:
If we never receive sentence expiry date in real life, why do we test against it? 🤔

This definitely seems a bit odd, so it's good to have this comment here. But it does make me wonder if we should just get rid of it if genuinely not used.